src/antsibull_build/changelog.py

```diff
@@ -717,6 +717,12 @@ def _get_removal_entry(  # noqa: C901, pylint:disable=too-many-branches
                 "community_steering_committee.html#creating-community-topic>`__."
             )
 
+    if sentences and reason not in ("renamed", "deprecated"):
+        sentences.append(
+            "After removal, users can still install this collection with "
```

Comment:
why switch from 'you can still install' to 'users can still install'?  Just a nit question really...

Reply:
I copied that sentence over from the deprecation/removal announcement text generation.

Reply:
https://github.com/ansible-community/antsibull-build/blob/9d8bfba28cf81f2b5e8025308bb5de82652723af/src/antsibull_build/changelog.py#L812-L816

Reply:
ok thanks for the explanation!